vital/types/point.h

```diff
@@ -69,6 +69,9 @@ class VITAL_EXPORT point
     m_covariance = v;
   }
 
+  T& operator[]( int i ) { return m_value[i]; }
+  const T& operator[]( int i ) const { return m_value[i]; }
```

Comment:
Considering the prior conversation, and also that `point` is often going to contain scalar types, I think this should probably return by value rather than by const-refernce...

Reply:
Ping?